en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md

```diff
@@ -1,361 +1,104 @@
+            Key Manager
         </td>
-    </tr>
-    <tr class="odd">
-    <td>Analytics Dashboard</td>
-    <td>Analytics Worker</td>
-    <td>9444</td>
-         <td>
-            Deploy business rules
-         </td>
-    </tr>
-    <tr class="even">
-    <td>Analytics Dashboard</td>
-    <td>Publisher</td>
-    <td>9443</td>
-         <td>
-            Use admin and publisher REST APIs
-         </td>
-    </tr>
-    <tr class="odd">
-    <td>Analytics Dashboard</td>
-    <td>Dev Portal</td>
-    <td>9443</td>
-         <td>
-            Use Dev-Portal portal REST APIs
-         </td>
-    </tr>
-    </tbody>
-    </table>
-    **Open port for Load balancer/User**
-    <table>
-    <thead>
-    <tr class="header">
-    <th>Server</th>
-    <th>Port</th>
-    <th>Description</th>
-    </tr>
-    </thead>
-    <tbody>
-    <tr class="odd">
-    <td>Key Manager</td>
-    <td>9443</td>
         <td>
-            Token endpoint and identity REST APIs/web apps
+            Responsible for all security and key-related operations.
         </td>
     </tr>
-    <tr class="even">
-    <td>Gateway</td>
-    <td>8243</td>
-        <td>
-            Gateway nio HTTPS
-        </td>
-     </tr>
-    <tr class="odd">
-    <td>Gateway</td>
-    <td>8280</td>
+    <tr>
         <td>
-            Gateway nio HTTP
+            Traffic Manager
         </td>
-    </tr>
-    <tr class="even">
-    <td>Gateway</td>
-    <td>8099</td>
         <td>
-            Gateway WS(Web socket)
+            Used to make a decision on throttling. It also works as an event hub for broadcasting controller events such as throttling events, block conditions, revoke token retrieval events, API events, API policy events, application events, application policy events, application keys events, subscription events, and subscription policy events.
         </td>
     </tr>
-    <tr class="odd">
-    <td>Gateway</td>
-    <td>9099</td>
+    <tr>
         <td>
-            Gateway WSS(Web socket secure)
+            Publisher
         </td>
-    </tr>
-    <tr class="even">
-    <td>Publisher</td>
-    <td>9443</td>
         <td>
-            Publisher portal, Admin and Publisher REST API
+            Enables API providers to easily publish their APIs, share documentation, provision API keys, and gather feedback on API features, quality, and usage.
         </td>
     </tr>
-    <tr class="odd">
-    <td>Dev Portal</td>
-    <td>9443</td>
+    <tr>
         <td>
-            Dev portal and Rest API
+            Developer Portal
         </td>
-    </tr>
-    <tr class="even">
-    <td>Analytics Dashboard</td>
-    <td>9643</td>
         <td>
-            Analytics dashboard and business rule portals
+            Enables consumers to self-register, discover API functionality, subscribe to APIs, evaluate them, and interact with API publishers.
         </td>
     </tr>
-    </tbody>
-    </table>
--->
\ No newline at end of file
+</table>
+
```

Comment:
Don't we add a diagram is this page on distributed setup with all the conections as in https://apim.docs.wso2.com/en/3.2.0/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m/ ?

Reply:
@isharac

Reply:
Will send a separate PR for this one when the diagrams are ready.